docs/resources/repository.md

```diff
@@ -141,9 +146,13 @@ Required:
 - `server_type` (String) Type of the MongoDB server. Allowed values:
   - `replicaset`
   - `standalone`
-  - `sharded`
+  - `sharded`.
+    The following conditions apply:
+  - If `server_type=standalone`, then only one `repo_node` block can be declared and it must be static. See `dynamic`. - If `server_type=sharded` and the a `srv_record_name` is provided, then all `repo_node` blocks must be declared dynamic, otherwise all of them must be static. See `dynamic`.
```

Comment:
The indentation of the second `- If` looks different than the first one, but this might just be my browser. Please check before merging.

Reply:
You are right. It is missing a new line.